### PR TITLE
README.md: Add link to Ansible collection creator path

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
README.md: Add link to Ansible collection creator path

### CODE REVIEW

**Review by felixfontein — APPROVED** [review]
Not sure whether this needs a vote, it is definitely better to wait for more approvals before merging.

**Review by mariolenz — APPROVED** [review]
LGTM

**Comment by Andersson007** [comment]
as most of folks involved in inclusion/exclusion are already here, merging:)
@felixfontein @mariolenz thanks for reviewing!

**Pull request merged**
